react-components/src/components/CacheProvider/types.ts

```diff
@@ -64,3 +65,13 @@ export type Image360AnnotationAssetInfo = {
 };
 
 export type AnnotationId = number;
+
+export type ChunkInCacheTypes = {
+  chunkInCache: Array<Required<AssetMapping3D>>;
+  chunkNotInCache: number[];
+};
+
+export type ChunkInCacheTypesNode3D = {
+  chunkInCache: Node3D[];
+  chunkNotInCache: number[];
+};
```

Comment:
Don't have a very strong opinion, but I think I'd prefer having one type that is generic here, e.g. 
```
export type CachedAndUncachedObjects<ObjectType> {
    cachedObjects: ObjectType[],
    uncachedIds: number[]
}
```

Reply:
I agree... make sense.

Changed here:
https://github.com/cognitedata/reveal/pull/4652/commits/01d9e4192802dde46298958dce3453c03f561cce